src/main/java/core/basesyntax/dp/Storage.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.dp;
+
+import java.util.HashMap;
+
+public class Storage {
+    public static final HashMap<String, Integer> storage = new HashMap<>();
```

Comment:
Using a static `HashMap` for storage can lead to concurrency issues if accessed by multiple threads simultaneously. Consider using a thread-safe collection like `ConcurrentHashMap` if your application will be multi-threaded. Additionally, static fields can make unit testing more challenging, as they maintain state across tests.